integration_tests/run.py

```diff
@@ -0,0 +1,239 @@
+# Copyright 2017 Google Inc.  All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Integration testing runner."""
+
+import argparse
+import json
+import multiprocessing
+import os
+
+from datetime import datetime
+from google.cloud import bigquery
+
+from googleapiclient import discovery
+from oauth2client.client import GoogleCredentials
+import time
+
+IMAGE_NAME = 'gcr.io/gcp-variant-transforms/gcp-variant-transforms'
+PIPELINE_NAME = 'gcp-variant-transforms-integration-test'
+SCOPES = ['https://www.googleapis.com/auth/bigquery']
+DEFAULT_ZONES = ['us-west1-b']
+SCRIPT_PATH = '/opt/gcp_variant_transforms/bin/vcf_to_bq'
+
+
+class TestCase(object):
+  """Test case that holds information to run in pipelines API.
+
+  To define a new integration test case, create a json file in
+  gcp_variant_transforms/integration_tests directory and specify at least
+  test_name, table_name, and input_pattern for the integration test.
+  """
+
+  def __init__(self,
+               context,
+               test_name,
+               table_name,
+               input_pattern,
+               **kwargs):
+
+    self._name = test_name
+    dataset_id = context.dataset_id
+    output_table = '{}:{}.{}'.format(context.project, dataset_id, table_name)
+    args = ['--input_pattern {}'.format(input_pattern),
+            '--output_table {}'.format(output_table),
+            '--project {}'.format(context.project),
+            '--staging_location {}'.format(context.staging_location),
+            '--temp_location {}'.format(context.temp_location),
+            '--job_name {}-{}'.format(test_name, dataset_id.replace('_', '-'))]
```

Comment:
Do you need to replace the '-' here? I think we should aim for not having it in the first place (since we generate the dataset id).